docs/releases/v2020.1.2.rst

```diff
@@ -0,0 +1,77 @@
+Gluon v2020.1.2
+===============
+
+Removed hardware support
+------------------------
+
+lantiq-xway
+~~~~~~~~~~~
+
+- AVM FRITZ!Box 7320
+- AVM FRITZ!Box 7330
+- AVM FRITZ!Box 7330 SL
```

Comment:
Linking the issue could be helpful. There was a proposed patch which nobody took interest in.